### PR TITLE
Don't float create/join team buttons over team rows.

### PR DESCRIPTION
**Pull request by adamjspooner**
@keybase/react-hackers, this is ready for review. It resolves this: https://github.com/keybase/client/issues/20980

### CODE REVIEW

File: shared/chat/inbox/container/index.tsx [hunk 1]
```diff
@@ -169,12 +171,19 @@ const Connected = Container.namedConnect(
...
       smallTeams = smallTeams.slice(0, stateProps.inboxNumSmallRows)
     }
-    const smallRows = makeSmallRows(smallTeams)
-    const bigRows = makeBigRows(bigTeams)
+    let smallRows = makeSmallRows(smallTeams)
+    let bigRows = makeBigRows(bigTeams)
+    const teamBuilder: Array<RowItemTeamBuilder> = [{type: 'teamBuilder'}]
+    if (smallRows.length !== 0) {
+      if (bigRows.length === 0) {
+        smallRows = [...smallRows, ...teamBuilder]
```
Comment (chrisnojima) [inline shared/chat/inbox/container/index.tsx]: change teamBuidler to just the RowItemTeamBuilder and just add it, you don't need to spread it
`smallRows = [...smallRows, teamBuilder]`
or just `smallRows.push(teamBuilder)`

**Comment by adamjspooner** [comment]
@keybase/react-hackers, bump.

**Comment by chrisnojima** [comment]
i previously approved

**Comment by adamjspooner** [comment]
🤦🏻‍♂️Sorry, I totally missed that.

**Pull request merged**